### PR TITLE
Move constant schedulerError in scheduler to v1 package

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

#### What type of PR is this?
/kind cleanup
/sig scheduling

#### What this PR does / why we need it:
Be consistent with another reason `v1.PodReasonUnschedulable`
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/lgtm

File: staging/src/k8s.io/api/core/v1/types.go [hunk 1]
```diff
@@ -2665,6 +2665,10 @@ const (
 	// PodReasonUnschedulable reason in PodScheduled PodCondition means that the scheduler
 	// can't schedule the pod right now, for example due to insufficient resources in the cluster.
 	PodReasonUnschedulable = "Unschedulable"
+
+	// PodReasonError reason in PodScheduled PodCondition means that some internal error happens
+	// during scheduling, for example due to nodeAffinity parsing errors.
+	PodReasonError = "SchedulerError"
```
Comment (alculquicondor) [inline staging/src/k8s.io/api/core/v1/types.go:2671]: I prefer

```suggestion
	PodReasonSchedulerError = "SchedulerError"
```

Comment (kerthcet) [reply staging/src/k8s.io/api/core/v1/types.go:2671]: Done

**Pull request merged**